### PR TITLE
Menu counters for metrics correlations

### PR DESCRIPTION
**Pull request by VLegakis**
PR's result with menu counters:
Screenshot 2020-08-27 at 9 14 47 PM

### CODE REVIEW

File: src/domains/dashboard/components/node-view/node-view.tsx [hunk 1]
```diff
@@ -152,7 +154,9 @@ export const NodeView = ({
     ((((width * pcentWidth) / 100) * chartsMetadata.update_every) / 3) / 60,
   ) * 60
 
-  const menus = useMemo(() => renderChartsAndMenu(chartsMetadata), [chartsMetadata])
+  const menus = useMemo(() => renderChartsAndMenu(metricsCorrelationMetadata
+    || chartsMetadata, metricsCorrelationMetadata
+    && chartsMetadata), [metricsCorrelationMetadata, chartsMetadata])
```
Comment (jacekkolasa) [inline src/domains/dashboard/components/node-view/node-view.tsx:159]: do you think that destructuring named arguments (ie. `renderChartsAndMenu({ chartsMetadata, metricsCorrelationMetadata })` would be more readable?

**Review by jacekkolasa — COMMENTED** [review]
good stuff, i understand it's work in progress, right? (draft pr).

File: src/domains/dashboard/components/node-view/node-view.tsx (hunk 1 above)
Comment (VLegakis) [reply src/domains/dashboard/components/node-view/node-view.tsx]: Cleaned up a bit this part. Please retake a look

**Review by jacekkolasa — APPROVED** [review]
lgtm! IMO it would be good someday to split ChartsMetadata from Metrics Correlation state (which are not metadata, ie. don't come from /charts endpoint). But right now it works, and from what i see it will be easy to clean it in the future.

**Pull request merged**